src/pudl/metadata/resources/eia861.py

```diff
@@ -276,17 +278,13 @@
     },
     "core_eia861__yearly_distributed_generation_misc": {
         # TODO: might want to rename this table to be _capacity
-        "description": (
-            """Information on the capacity of utility or customer-owned distributed
-generation. Includes the number of generators, whether the capacity is estimated or
-actual, the amount of backup capacity, and how much capacity is from generators with
-less than 1 MW of nameplate capacity.
-
-The raw EIA861 distributed generation (DG) table (split into three normalized tables in
+        "description": {
+            "additional_summary_text": "the capacity and quantity of utility or customer-owned distributed generation",
```

Comment:
Sneaky missing period
```suggestion
            "additional_summary_text": "the capacity and quantity of utility or customer-owned distributed generation.",
```